Formula/aircrack-ng.rb

```diff
@@ -1,13 +1,16 @@
 class AircrackNg < Formula
   desc "Next-generation aircrack with lots of new features"
   homepage "https://aircrack-ng.org/"
-
-  # We can't update this due to linux-only dependencies in >1.1.
-  # See https://github.com/Homebrew/homebrew/issues/29450
-  url "https://download.aircrack-ng.org/aircrack-ng-1.1.tar.gz"
-  sha256 "b136b549b7d2a2751c21793100075ea43b28de9af4c1969508bb95bcc92224ad"
   revision 2
 
+  stable do
+    url "https://download.aircrack-ng.org/aircrack-ng-1.1.tar.gz"
+    sha256 "b136b549b7d2a2751c21793100075ea43b28de9af4c1969508bb95bcc92224ad"
+    # Remove root requirement from OUI update script. See:
+    # https://github.com/Homebrew/homebrew/pull/12755
+    patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. I realise you didn't create it but if you could submit it that'd be great. Thanks!